.github/workflows/pre-commit.yml

```
@@ -24,3 +35,6 @@ jobs:
           path: ~/.cache/pre-commit
           key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
       - uses: pre-commit/action@v3.0.1
+        with:
+          files: ${{ github.event.inputs.files }}
```

actionlint (1.7.4)

property "files" is not defined in object type {}

(expression)